crowbar_framework/app/models/node_object.rb

```diff
@@ -1329,6 +1329,22 @@ def set_state(state)
     result
   end
 
+  def actions
+    [
+      "allocate",
+      "delete",
+      "identify",
```

Comment:
I know you are just submitting @vwallfahrer code here, but can you somehow split this list in two, so that the `machines_controller` can use it as well?
that would be
```ruby
  [
    :update,
    :identify
  ]
```
and
```ruby
[
    :reinstall,
    :reset,
    :shutdown,
    :reboot,
    :poweron,
    :powercycle,
    :poweroff,
    :allocate,
    :delete,
    :identify
  ]
```
and merge them when you need it in the `nodes_controller`
then we would have saved some duplicated code.

https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/app/controllers/machines_controller.rb#L126
and
https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/app/controllers/machines_controller.rb#L105